.gitignore

```diff
@@ -2,4 +2,5 @@
 *.swp
 /docs/fidelity_bonds.md
 Cargo.lock
-.vscode
\ No newline at end of file
+.vscode
+.idea
```

Comment:
Whats a .idea?

Reply:
I started using a new IDE called Rust Rover. It creates a .idea module, similar to how VS Code creates a .vscode module.

Reply:
Cool..